packages/beacon-node/src/chain/seenCache/seenGossipBlockInput.ts

```diff
@@ -26,8 +26,8 @@ export type BlockInputCacheType = {
 };
 
 const MAX_GOSSIPINPUT_CACHE = 5;
-// ssz.deneb.BlobSidecars.elementType.fixedSize, 131256 is size for mainnet preset;
-const BLOBSIDECAR_FIXED_SIZE = ssz.deneb.BlobSidecars.elementType.fixedSize ?? 131256;
+// ssz.deneb.BlobSidecars.elementType.fixedSize, 131928 is size for mainnet preset;
+const BLOBSIDECAR_FIXED_SIZE = ssz.deneb.BlobSidecars.elementType.fixedSize ?? 131928;
```

Comment:
Just set it to a constant value and here and assert it's correct in a unit test, or wrap in a function that throws if null